Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -196,6 +196,11 @@ class BrowserViewController: UIViewController {
         let contextViewModel = ContextualHintViewModel(forHintType: .toolbarLocation,
                                                        with: profile)
         self.contextHintVC = ContextualHintViewController(with: contextViewModel)
+        
+        // Ecosia: User's stored `apnConsentReminderModel` initialization
+        if User.shared.apnConsentReminderModel == nil {
+            User.shared.apnConsentReminderModel = BrowserViewController.userApnConsentOptInModel
+        }
```

Comment:
If we need to guarantee `User.shared` has a non-nil `apnConsentReminderModel`, how about changing it [inside the User class](https://github.com/ecosia/ios-core/blob/main/Sources/User.swift#L48) and setting a default value?

Reply:
How could I not think about that! Thanks will apply.
Really 🤔.
Can't think of why

Reply:
Now I remember, 😅 .
The original implementation wanted the OptInManager to create the model only if needed.
Now that the OptInManager leaves within the BrowserViewController instance, we'll have it anyway and would be a bit over overeng to make it only in case we've got the EngagementService enabled.
WDYT?

Reply:
I updated the PR given the change done that reflects what's shared above. 
Let me know if you agree in case 👌

Reply:
Yeah, agreed, even if it's only relevant for a small percentage of users, it would be more complex to add it only in some cases

Reply:
Will remove the draft then. Thanks for the quick double-check!